RevenueCat/Scripts/Purchases.cs

```diff
@@ -86,6 +80,23 @@ public partial class Purchases : MonoBehaviour
              "NOTE: This value will be ignored if \"Use Runtime Setup\" is true. For Runtime Setup, you can configure " +
              "it through PurchasesConfiguration instead")]
     public string proxyURL;
+    
+    [Header("⚠️ Deprecated")]
+    [Tooltip("⚠️ RevenueCat currently uses StoreKit 1 for purchases, as its stability in production " +
+             "scenarios has proven to be more performant than StoreKit 2.\n" +
+             "We're collecting more data on the best approach, but StoreKit 1 vs StoreKit 2 is \n" +
+             "an implementation detail that you shouldn't need to care about.\n" +
+             "We recommend not using this parameter, letting RevenueCat decide for " +
+             "you which StoreKit implementation to use.\n" +
+             "NOTE: This value will be ignored if \"Use Runtime Setup\" is true. For Runtime Setup, you can configure " +
+             "it through PurchasesConfiguration instead")]
```

Comment:
this text turns into this in the Unity Editor:
<img width="619" alt="Untitled" src="https://user-images.githubusercontent.com/3922667/224795531-b9ed0238-0219-4be7-a8e5-0d67471c264f.png">